content/sensu-go/5.9/getting-started/get-started.md

```diff
@@ -10,28 +10,36 @@ menu:
     parent: getting-started
 ---
 
-### Try the sandbox
+### Learn Sensu in 15 minutes
 
-The sandbox is the best place to get started with Sensu and try out new features.
+Create your first monitoring event pipeline using a local development environment pre-installed with the essential Sensu stack.
 
 - [Download the sandbox and learn Sensu Go][7]
 - [See more sandbox lessons][1]
+- [Join the community][8]
 
-### Install Sensu Go
+### Install Sensu Go (free tier)
```

Comment:
"free tier" is 👌 ... if you feel the need to define that somewhere, perhaps we can work on some copy that explains the free tier as the official/tested binaries and installer packages available at sensu.io/downloads. The "free tier" documentation should also apply to "versions of Sensu Go that are compiled from source (https://github.com/sensu/sensu-go), except where noted"; e.g. we can add footnotes to features like our homepage to indicate that it is included in the free tier, but not included in the MIT-licensed source code. 

*IF* we find it necessary to provide such a definition, I would expect we would only include it in the documentation, and not the marketing site. The feature matrix at https://sensu.io/products should suffice for that. 

/cc @bananabele

Reply:
Definitely worth thinking about! And agree that the feature matrix on /products has enough info for the marketing site.